src/alcotest-lwt/dune

```diff
@@ -7,5 +7,4 @@
   fmt
   alcotest.engine
   alcotest
-  (re_export lwt.unix) ;; Unused dependency added for convenience to consumers
```

Comment:
This looks like a breaking change.. I don't know how many packages depend on lwt.unix being re-exported though.